GSA-TTS/federal-website-standards
# Visual cleanup

## Context
Fixes:
- #231 
- #234 
- #237 

## Description
* Visual fixes related to the recent CSS change that shrunk the `width` of `grid-container`
* Undo some overrides to the style of the USWDS step indicator we're using on each of the standard pages so that the style now matches default styling of the component
*  Adds a step indicator to the standards overview page
* The "Federal website standards" text in the header was wrapping to two lines and should now never wrap
* Renamed `_includes/status/standard.html` to `_includes/status/required.html` to match the status name since the final step was recently changed from "Standard" to "Required"

## How to verify this change
1. Visit the [preview standards page](https://federalist-142078a8-f654-4daa-8f73-db9770b3ec7a.sites.pages.cloud.gov/preview/gsa-tts/federal-website-standards/caley/cleanup-round-2/standards/) and verify that the step indicator is there
2. Padding on the standards overview page has been tweaked, @evoeges can double check
3. Visit a standard from each status and verify that the USWDS step indicator no longer has a grey background, no longer has padding, and that the white or grey stroke around each step number is present again. You can use these pages: [HTML Page Title](https://federalist-142078a8-f654-4daa-8f73-db9770b3ec7a.sites.pages.cloud.gov/preview/gsa-tts/federal-website-standards/caley/cleanup-round-2/standards/html-page-title/) (Pending), [Site Search](https://federalist-142078a8-f654-4daa-8f73-db9770b3ec7a.sites.pages.cloud.gov/preview/gsa-tts/federal-website-standards/caley/cleanup-round-2/standards/site-search-draft/) (Draft), [Contact](https://federalist-142078a8-f654-4daa-8f73-db9770b3ec7a.sites.pages.cloud.gov/preview/gsa-tts/federal-website-standards/caley/cleanup-round-2/standards/contact-research/) (Research)
4. On any page: Verify that the header text "Federal website standards" is not wrapping to two lines on desktop viewports (>1023px wide)

## Discussion
We need to sort out what text to put above the step indicator that explains that there are 4 steps in the process. We did have "The process of becoming a standard has 4 steps." prior to the content changes from OMB being implemented. We could go with that again but we're brainstorming it now.
